pyproject.toml

```diff
@@ -12,21 +60,21 @@ warn_no_return = false
 [tool.black]
 # https://github.com/psf/black
 line-length = 120
-target-version = ['py37']
 
 
 [tool.ruff]
 line-length = 120
-target-version = "py37"
 # Enable Pyflakes `E` and `F` codes by default.
 select = [
-    "E", "W",  # see: https://pypi.org/project/pycodestyle
-    "F",  # see: https://pypi.org/project/pyflakes
-    "I",  # see: isort
-]
-ignore = [
-    "E501",
-    "E741",
+    "E",
+    "W", # see: https://pypi.org/project/pycodestyle
+    "F", # see: https://pypi.org/project/pyflakes
+    "I", # see: isort
 ]
+ignore = ["E501", "E741"]
 ignore-init-module-imports = true
 unfixable = ["F401"]
+
+[tool.pytest.ini_options]
+python_files = 'tests/*.py'
+addopts = "--color=yes"
```

Comment:
The rest of the file uses double quotes.

Reply:
I don't think this matter...

Reply:
Maybe it doesn't _matter_, but it's always good to aim for consistency.